src/lib/flows/claim-project-flow/steps/review/review.svelte

```diff
@@ -250,9 +250,7 @@
           <span class="typo-text-bold">Drips dashboard</span>.</UlIconLi
         >
         <UlIconLi icon={TokenStreamsIcon}
-          >Future incoming funds will be split to your recipients <span class="typo-text-bold"
-            >monthly</span
-          >, on the last Thursday of every month.</UlIconLi
+          >{@html network.settlement.recipientsExplainerHtml}</UlIconLi
```

Comment:
We got burned once by `@html` resulting in what some might call a JS injection vulnerability 😇 since then, always feels better to wrap anything we pass to `@html` in `sanitize`. There's already a function for that available. I know in this case nothing user-supplied is being rendered, but better safe than sorry.